pygmt/datasets/load_remote_dataset.py

```diff
@@ -581,22 +579,9 @@ def _load_remote_dataset(
         raise GMTInvalidInput(msg)
 
     fname = f"@{prefix}_{resolution}_{reg}"
```

Comment:
I see a lot of error messages like:
```
Error: h [ERROR]: Tile @S90W180.earth_age_01m_g.nc not found!
Error: h [ERROR]: Tile @S90W150.earth_age_01m_g.nc not found!
Error: h [ERROR]: Tile @S90W120.earth_age_01m_g.nc not found!
Error: h [ERROR]: Tile @S90W090.earth_age_01m_g.nc not found!
Error: h [ERROR]: Tile @S90W060.earth_age_01m_g.nc not found!
Error: h [ERROR]: Tile @S90W030.earth_age_01m_g.nc not found!
Error: h [ERROR]: Tile @S90E000.earth_age_01m_g.nc not found!
Error: h [ERROR]: Tile @S90E030.earth_age_01m_g.nc not found!
```
This is because, in the GMT backend, we use something like `which("@earth_age_01m_g")` to get the file path, which doesn't work well for tiled grids.

Reply:
Yeah, we used to do this:

```python
    # Full path to the grid if not tiled grids.
    source = which(fname, download="a") if not resinfo.tiled else None
    # Manually add source to xarray.DataArray encoding to make the GMT accessors work.
    if source:
        grid.encoding["source"] = source
```

i.e. only add the source for non-tiled grids, so that the accessor's `which` call doesn't report this error. I'm thinking if it's possible to either 1) silence the `which` call (does `verbose="q"` work?), or 2) add some heuristic/logic to determine whether the source is a tiled grid before calling `which` in `GMTBackendEntrypoint`

Reply:
> I'm thinking if it's possible to either 1) silence the `which` call (does `verbose="q"` work?), or 2) add some heuristic/logic to determine whether the source is a tiled grid before calling `which` in `GMTBackendEntrypoint`

I think either works. Perhaps `verbose="q"` is easier?

Reply:
Done in commit 5557b3343247dd1a5312b0ff921daf6f06d56d9d